### PR TITLE
Feature/date polish

### PR DESCRIPTION
**Pull request by sebastian-espeset**
## Changes
-adds new date object to send to data base for the sake of consistency 
-polishes up ui

# Clearly document the motivation and what changed
 - [x] a clear rationale for the feature (user story)
 - [ ] a screen shot showing either the intended state or completed state of the feature
 - [x] additional context about changes made in the PR 

# Atomic, well named commits
 - [x] name is specfic to feature changes or additions
 - [x] commits have descriptive messages regarding what changed in the code
  
# Appropriately scoped  
- [x] focused on a single problem
- [x] small enough/ easy to review

# Adequately reviewed
- [x] sent to multiple reviewers
- [x] reviewed by at least two people
- [ ] cross team review if necessary 
- [x] ensure there is at least one piece of feedback from each reviewer

# Code is clear and maintainable
- [x] consistently formatted
- [x] has config files to enforce linting and formatting
- [x] NO LOGS OR PRINT STATEMENTS
- [x] code is idiomatic/best practice  

- [ ] No failing unit test

### CODE REVIEW

File: api/upload/uploadRouter.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const router = express.Router();
 const Upload = require('./uploadModel');
 require('dotenv').config();
 const authRequired = require('../middleware/authRequired');
+const { date } = require('faker');
```
Comment (JonathanMary) [inline api/upload/uploadRouter.js]: Let's delete this.

**Comment by JonathanMary** [comment]
We will need to commit the change to "data\migrations\20201202194136_cases.js" file.
From ` table.string('date');` to `table.date('date');` on line 13.

File: api/upload/uploadRouter.js (hunk 1 above)
Comment (dgamboa) [inline api/upload/uploadRouter.js]: Why do we need to import `date` from `faker`?

**Review by dgamboa — COMMENTED** [review]
I think importing `date` from `faker` may have been an auto-populate from VS Code. Looks like we don't use it so we should remove that line. Also, does creating a date object from the scraped date always work? If so, would it be better to pass the string directly to the database and just change the date column on the database to be `type: date`?

**Review by JonathanMary — COMMENTED** [review]
> I think importing `date` from `faker` may have been an auto-populate from VS Code. Looks like we don't use it so we should remove that line. Also, does creating a date object from the scraped date always work? If so, would it be better to pass the string directly to the database and just change the date column on the database to be `type: date`?

You're correct. Maybe we merge this and do the deleting now?

File: api/upload/uploadRouter.js (hunk 1 above)
Comment (sebastian-espeset) [reply api/upload/uploadRouter.js]: Sounds great! I saw this last night and was curious about it.

Comment (JonathanMary) [reply api/upload/uploadRouter.js]: removed it

**Pull request merged**